### ISSUE TITLE
Wysihat bug on IE8: toolbar can not display proper

### ISSUE THREAD

**Issue by dukec2020** [80beans/wysihat-engine#6]
I've installed wysihat-example-demo. It work well on FF,IE6,IE7 but not IE8.
The icon in toolbar can't display - only alter text is displayed.
Here is error's screenshot on IE8, script debug.

http://picasaweb.google.com/lh/sredir?uname=hvcong&target=PHOTO&id=5419228662312599314&aid=5281051008551701777&authkey=Gv1sRgCK32wvqs-8Wm8gE&feat=email

**Comment by joselo** [comment]
yes in IE8 doesn't apper the buttons, do you have some solution??

**Comment by umbalaconmeogia** [comment]
A work around: add a special HTML tag to each document, right after the &lt;head&gt; tag
&lt;meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" /&gt;

Reference http://www.swiftcore.com/en/software/ie8-compatibility-make-ie8-render-same-ie7.html

**Comment by jeffkreeftmeijer** [comment]
Thanks for pointing this out, guys! I'm pretty sure this is a problem in the engine, so I'll go check it out. There are some nasty IE8 bugs in the WysiHat core though, so it could be that too. 

If you have any ideas, please be my guest. I'll keep you posted :)
- Jeff

**Comment by walterdavis** [comment]
Any progress on this front? I just hit this bug, and while the fix above does work, it also means that I can't rely on all the other good things that changed between IE7 and 8.

Thanks,

Walter

**Comment by paquettej** [comment]
I've hit this as well... any update?

**Comment by walterdavis** [comment]
I haven't heard a peep...
